README.md

```diff
@@ -282,7 +282,7 @@ I would like to thank...
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT) licensed.
```

Comment:
- [ ] When I clicked the MIT link see the error I got. I think you can fix this by appending `.md` to the link.

  ```
  ...(./MIT.md)...
  ```

  ![cjcj](https://user-images.githubusercontent.com/54848363/208648559-94350354-cf51-4e49-9b87-7c4315588738.jpg)